dashboard/src/main/kotlin/com/ritense/dashboard/web/rest/dto/DashboardWidgetDataResultDto.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.dashboard.web.rest.dto
+
+import com.fasterxml.jackson.annotation.JsonUnwrapped
+
+class DashboardWidgetDataResultDto(
+    val key: String,
+    @JsonUnwrapped val result: Any
```

Comment:
i'm not sure if i like that this is @JsonUnwrapped. it feels like it would be better to put the object with unknown content in it's own variable.

Reply:
I'm was just following the spec. If we want to change that, we need to change the spec.

Reply:
changed it.